types/node/fs.d.ts

```diff
@@ -260,6 +260,29 @@ declare module 'fs' {
          */
         readSync(): Dirent | null;
     }
+    /**
+     * Class: fs.StatWatcher
+     * @since v14.3.0, v12.20.0
+     * Extends `EventEmitter`
+     * A successful call to {@link watchFile} method will return a new fs.StatWatcher object.
+     */
+    export interface StatWatcher extends EventEmitter {
+        /**
```

Comment:
Also sorry I missed this, what are the available events on this emitter?

Reply:
Good question -- from [the documentation](https://nodejs.org/api/fs.html#class-fsstatwatcher) it doesn't look like there are any on the base class.

Reply:
[Looking at the implementation of `StatWatcher` in node](https://github.com/nodejs/node/blob/340b770d3f4c3b50a5d36c0776afd63d9b4f22de/lib/internal/fs/watchers.js#L68), it also doesn't look like there are any events on this emitter.